hs_access_control/models.py

```diff
@@ -2262,6 +2278,9 @@ def get_combined_privilege(self, this_user):
             raise HSAUsageException("Grantee is not a user")
         access_user = this_user.uaccess
 
+        if this_user.is_superuser:
+            return PrivilegeCodes.OWNER
+
```

Comment:
I think the superuser (admin) does not have uaccess back-relation attribute since admin is created by default by Django without going through our create_account() function where UserAccess relation is created with the new user. So the statement before this addition, i.e., access_user = this_user.uaccess, should be moved after is_superuser conditional check statement.